pkg/apis/networking/v1/types.go

```diff
@@ -108,4 +108,16 @@ type BGPPeer struct {
 	Password string `json:"password,omitempty"`
 }
 
+type VTEPInfo struct {
```

Comment:
This struct is both used by `networking/v1` and `multicluster/v1`, so is it better to place it on `pkg/apis/common/types.go`?

Reply:
Let's do it in another PR.